src/styles/_stars.scss

```diff
@@ -0,0 +1,35 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 16px;
+  width: 100%;
+  margin-top: 16px;
+
+  &__list {
+    @include center;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    justify-content: space-between;
```

Comment:
The `justify-content` property is used here, but it seems the `.stars__star` class is not defined as a flex container. Ensure that `display: flex;` is applied to the `.stars__star` class if you intend to use flex properties.